src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
I'd like to see another command that does fine control of the x,y position with the joysticks

Reply:
Thats what armcommand does right now, state control is done by the subsystem

Reply:
nevermind, adding a new command to change positions